runner/runners.go

```diff
@@ -404,19 +422,43 @@ func AddDefaultPath(args []string, defaultMemberPath string) []string {
 	return append(args, fmt.Sprintf("--path=%s", defaultMemberPath))
 }
 
-// AddDefaultTargetForTiny will add --target=x86_64-unknown-linux-musl if the stack is Tiny and a `--target` is not already set
-func AddDefaultTargetForTiny(args []string, stack string) []string {
-	if stack != libpak.TinyStackID {
-		return args
+// AddDefaultTargetForTinyOrStatic will add the appropriate options if not already set
+func AddDefaultTargetForTinyOrStatic(args []string, stack string, staticType string) ([]string, error) {
+	if !libpak.IsTinyStack(stack) && !libpak.IsStaticStack(stack) {
+		return args, nil
 	}
 
+	// user already picked a target, back off
 	for _, arg := range args {
 		if arg == "--target" || strings.HasPrefix(arg, "--target=") {
-			return args
+			return args, nil
+		}
+	}
+
+	// user set flags to do a static build, back off
+	rustFlags := os.Getenv("RUSTFLAGS")
+	if strings.Contains(rustFlags, "target-feature=+crt-static") {
```

Comment:
Is this assumption here that a user who has set `RUSTFLAGS=target-feature=+crt-static` will explicitly set their own target also?

It seems to me that it would be a better UX if we set the target as well rather than relying on the user to set two things. But maybe I'm missing something.

Reply:
Sorry btw. I had posted the previous comment on my phone not realizing it was never actually published.